### ISSUE TITLE
add audit-ID to issue API

### ISSUE THREAD

**Issue by mikehelmick** [google/exposure-notifications-verification-server#1076]
### TL;DR

Add an `auditID` field to the IssueCodeRequest API. This should be ignored via the web UI (the logged in user is the auditID).

This auditID should be used to calculate the current realm statistics alongside actual logged in users.

The use case for this is a 3rd party EHR type integration where they want to (on their side) track which physicians have issued codes by assigning a unique ID for that purpose, and pass that unique ID.

Then if a realm administrator sees that API user `123-abc-456` has issued a bunch of codes, they can know which provider has the API key associated with that audit ID and can go to that EHR provider to find who they physician is.

Needs work on the
* Issue API
* stats collection
* stats display + export

/kind enhancement
